android/build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         classpath 'com.android.tools.build:gradle:3.5.0'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         // Uncomment the line if you want to use firebase
-        // classpath 'com.google.gms:google-services:4.3.3'
+        classpath 'com.google.gms:google-services:4.3.3'
```

Comment:
This is an optional step and should not be uncommented by default

Reply:
Yeah got that.. It was uncommented while I was trying to test Firestore. Have forgotten to comment out again...

Reply:
@im-Amitto since it is necessary to have Firebase configured in the project for all the functions to work properly, can we keep uncommented all the Firebase related codes by default?

Otherwise we need to comment them again if we need to do a PR which seems to be little inconvenient.. :thinking: